src/NATS.Client.JetStream/NatsJSContext.Streams.cs

```diff
@@ -15,7 +15,7 @@ public partial class NatsJSContext
     /// <exception cref="NatsJSException">There was an issue retrieving the response.</exception>
     /// <exception cref="NatsJSApiException">Server responded with an error.</exception>
     public ValueTask<NatsJSStream> CreateStreamAsync(string stream, string[] subjects, CancellationToken cancellationToken = default) =>
-        CreateStreamAsync(new StreamCreateRequest { Name = stream, Subjects = subjects }, cancellationToken);
+        CreateStreamAsync(new StreamCreateRequest {Name = stream, Subjects = subjects}, cancellationToken);
```

Comment:
Please could you run `dotnet format` with your last commit and make sure there are no build warnings? Avoids having style changes polluting the diffs.

Reply:
done